python/strings_udf/strings_udf/_lib/cudf_jit_udf.pyx

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
+#
+# distutils: language = c++
+# cython: c_string_type=unicode, c_string_encoding=utf8
+
+import os
+
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+from libcpp.utility cimport move
+from libcpp.vector cimport vector
+
+from cudf.core.buffer import Buffer
+
+from cudf._lib.column cimport Column
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.column.column_view cimport column_view
+from cudf._lib.cpp.types cimport size_type
+from rmm._lib.device_buffer cimport DeviceBuffer, device_buffer
+
+from strings_udf._lib.cpp.strings_udf cimport (
+    to_string_view_array as cpp_to_string_view_array,
+)
+
+import numpy as np
+
+
+def to_string_view_array(Column strings_col):
+    cdef unique_ptr[device_buffer] c_buffer
+
+    # with nogil:
```

Comment:
Should this be commented out?

Reply:
Fixed this both here and in `strings_udf.pxd`